README.md

```diff
@@ -14,15 +17,38 @@ The dataset consists of 37.4 million interactions, |U| ≈ 2.3) million  users a
 FINN.no is the leading marketplace in the Norwegian classifieds market and provides users with a platform to buy and sell general merchandise, cars, real estate, as well as house rentals and job offerings.
 For questions, email simen.eide@finn.no or file an issue.
 
+## Install
+
+`pip install recsys_slates_dataset`
+
+## How to use
+
+To download the generic numpy data files:
+
+```python
+from recsys_slates_dataset import datahelper
+datahelper.download_data_files()
+```
+
+
+
+
+    True
+
```

Comment:
maybe change to `datahelper.download_data_files(True)`

Reply:
nice to have: A little [tqdm bar](https://tqdm.github.io/) to see how the download progresses?